examples/nlp/language_modeling/megatron_gpt_pretraining.py

```diff
@@ -63,10 +62,6 @@ def main(cfg) -> None:
 
     trainer = Trainer(plugins=plugins, **cfg.trainer)
 
-    # NLPDataConnector used to provide global batches which are needed
-    # for Apex fwd/bwd functions
-    trainer._data_connector = NLPDataConnector(trainer)
```

Comment:
Can you totally remove `NLPDataConnector` ? I mean remove it from nlp_overrides.py as well.

Reply:
done in https://github.com/NVIDIA/NeMo/pull/3665/commits/94e8d0c48a0db199500534ea47ae2e62518a3399